src/visGeometry/GeometryStore.ts

```diff
@@ -408,16 +433,25 @@ class GeometryStore {
         this.mlogger.debug(`Geo for id ${id} set to '${url}'`);
         const isMesh = displayType === GeometryDisplayType.OBJ;
         const isPDB = displayType === GeometryDisplayType.PDB;
-        if (!url) {
+        if (!url || (!isMesh && !isPDB)) {
```

Comment:
is this if statement even needed here? 
Do we even need to check for url if displayType is CUBE or SPHERE? 
If not, then you could just do `if (!url && displayType===GeometryDisplayType.CUBE)` and likewise for `SPHERE` without the outer if statement.

Reply:
I tried to make this clear in the function I just wrote, but it's checking for the case where someone selected "OBJ" or "PDB" but forgot to include a url.